### ISSUE TITLE
Ascii outline table missing on Android Cell

### ISSUE THREAD

**Issue by rgleason** [opencpn-manuals/development#48]
Describe the bug
Android cell phones do not handle AsciiDr outline table at all well.
Therefore the Dev manual is unreachable.
I think there should be a direct link to the Dev manual in the text or some other effective way to navigate to and through the Dev manual.

Perhaps this is just a deficiency we must live with, but I am finding the links to the dev manual and plugins very circular on this cell phone.

To Reproduce

    Browse to Developer Manual
    No Navigation on the left
    Unable to navigate and drill down.

Expected behavior
Ability to navigate properly.

Smartphone/Tablet (please complete the following information if applicable):

    Device: Cell Phone Android
    Andorid version: 13

**Comment by rgleason** [comment]
I agree with you there, not worth it.  I will try search.  Don't like using phone anyway.

**Comment by leamas** [comment]
>  Don't like using phone anyway.

Who does? 

Shall we close this?

**Comment by rgleason** [comment]
Just a thought after trying search. The trouble with search is it is totally blind.
With the OpenCPN Dokuwiki Manual what happens on a cell phone is there is a  button shown called "Sidebar"
Which opens up the entire Navigation Outline Table.

![Dokuwiki-Sidebar](https://github.com/opencpn-manuals/development/assets/2363959/b91940ad-243f-458e-951f-1d0721eb044c)

**Comment by rgleason** [comment]
https://docs.asciidoctor.org/asciidoc/latest/toc/

https://docs.asciidoctor.org/pdf-converter/latest/toc/

I think this can be one line on the page maybe just showing the top sections or just two levels down.

**Comment by leamas** [comment]
asciidoctor has nothing to do with this, the overall UI is generated by Antora. More precise, the forked javascript code at  https://gitlab.com/leamas/antora-ui-default 

But did we not agree not to spend  time on this?

**Comment by rgleason** [comment]
I think we did.  Perhaps some thing for the future, if we find some code that applies?

**Comment by rgleason** [comment]
We have the navigation file nav.adoc   https://docs.antora.org/antora/latest/navigation/files-and-lists/
We just need a link to it in the body that is not expanded, such that taping on it expands it.

Asked a questions on antora  zulip  https://antora.zulipchat.com/#narrow/stream/282404-community-.F0.9F.91.A5/topic/Link.20to.20nav.2Eadoc.20in.20a.20page.20for.20android.20Cell.20phone

**Comment by rgleason** [comment]
Please see  https://antora.zulipchat.com/#narrow/stream/282400-users/topic/Link.20to.20nav.2Eadoc.20in.20a.20page.20for.20android.20Cell.20phone

ESkwayrd writes 
2:56 PM

> There is no "navigation page", at least not with the Antora Default UI. The primary site navigation is included on every page in a vertical pane on the left. For small viewports, that pane is hidden by default and is accessible using the hamburger menu in the page header.
> 
> Linking to nav.adoc won't work:
> 
>     it's not a page, it's a file containing a list that is used by Antora's navigation infrastructure.
>     it doesn't exist in the pages folder, so an xref or link cannot find it.
> 
> However, there is a way:
> 
>     Create your .adoc to include a partial where the "real" navigation lives.
>     Create the partial with the navigation that should be included in nav.adoc and also be usable in a page.
>     In the page where navigation should be visible, include the partial.
> 
> Let me know if you need working examples.
> 
> Note that you should make all of the xrefs in the partial into absolute xrefs. Relative xrefs won't work when the partial is included at differing depths in the file structure. At the very least, include the module name in the xref. For example: * xref:module:page.adoc[]. If you include the partial across components, the partial has to include the component name too.
>

**Comment by rgleason** [comment]
After digging into this further, it should work properly. The plugins work properly, the Development Manual does not. It does not even work properly on a computer.

https://opencpn-manuals.github.io/main/ocpn-dev-manual/0.1/intro-AboutThisManual.html


NOTE: The plugins link works properly.  The TOC is found at the bottom of the page!
The short TOC is inserted at the top.
https://opencpn-manuals.github.io/main/ocpn-dev-manual/index.html

Ocpn-Dev-Manual
![Screenshot_20230804_203549_Firefox](https://github.com/opencpn-manuals/development/assets/2363959/4ff3ff56-0495-472a-8db1-0e5b76016bdf)

Plugins Manual
![Screenshot_20230804_203606_Firefox](https://github.com/opencpn-manuals/development/assets/2363959/454da6a2-848e-40cc-9085-b105bd46c2f2)

Main Plugins
![Screenshot_20230804_203625_Firefox(1)](https://github.com/opencpn-manuals/development/assets/2363959/43286c4f-a342-492a-9e15-2aa891148b1a)

Plugin Catagories
![Screenshot_20230804_203633_Firefox](https://github.com/opencpn-manuals/development/assets/2363959/2e000483-cf36-43cb-84ec-a4be970cb099)

![Screenshot_20230804_203643_Firefox](https://github.com/opencpn-manuals/development/assets/2363959/f04a2aa6-c0dc-495b-956c-da7b7ec66f7d)

-----
NOTE: The Developer Manual does NOT work properly. No TOC is found at the bottom of the page.
https://opencpn-manuals.github.io/main/ocpn-dev-manual/0.1/intro-AboutThisManual.html

About this Manual
I cannot take a screenshot due to "security policy"  --What is this about?
This page has some of the short TOC at the top, but it leads to a link that does not work on the next page  when you hit "Developer Manual" at the bottom.

This is horrendously messed up.  It should work.
Also
